### DISCUSSION TITLE
Can I change my world to Master Mode?

### DISCUSSION THREAD

**Discussion by paulapp** [Pryaxis/TShock#1753] [Q&A] [answered]
Hi there, new here. Was wondering if it's possible to change an existing Expert world to Master. Is it possible yet? Thanks.

**Comment by Kojirremer** [comment, marked as answer]
The 1.4 build of TShock is currently being developed, so not yet. Once it's available, it may (probably) have a similar way to toggle master mode as how it currently toggles expert.

**Comment by QuiCM** [comment]
I've added the command /worldmode in the latest prerelease (find it [here](https://github.com/Pryaxis/TShock/releases/tag/v4.4.0-pre2)). You should be able to use `/worldmode master` to change - no idea how well it will work though as we're entering unknown territory!

**Reply by DeadlyV** [reply]
/worldmode master or /worldmode 3 changes my world from expert to journey

**Reply by QuiCM** [reply]
Yeah i did a stupid. `/worldmode expert` will give you master. My bad! :(
